src/ompl/base/PlannerTerminationCondition.h

```diff
@@ -120,6 +121,32 @@ namespace ompl
 
         /** \brief Return a termination condition that will become true \e duration seconds in the future (wall-time), but is checked in a separate thread, every \e interval seconds; \e interval must be less than \e duration */
         PlannerTerminationCondition timedPlannerTerminationCondition(double duration, double interval);
+
+        /** \brief Return a termination condition that will become true as soon as the problem definition has an exact solution */
+        PlannerTerminationCondition exactSolnPlannerTerminationCondition(ompl::base::ProblemDefinitionPtr pdef);
+
+        /** \brief A class to run a planner for a specific number of iterations. Casts to a PTC for use with Planner::solve */
+        class IterationTerminationCondition
```

Comment:
The () operator returns a PTC. So I used it like:

```C++
IterationTerminationCondition myPTC(1u)
ompl::base::Planner myPlanner;
... //Setup problem, planner etc.
while (...)
{
    myPlanner.solve(myPTC); //Run planner for 1 call to the PTC
    ... //Save planner status to file, etc.
    myPTC.reset(); //Reset the PTC to allow another iteration
}
```
I think I'm commenting concurrently, so I will stop for now, but I can move it anywhere you want (including /dev/null :wink:).